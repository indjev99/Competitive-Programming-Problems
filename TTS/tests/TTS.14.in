30 7
9212.87 -3888.83
2939.72 -9558.14
-9182.22 -3960.66
-3489.62 -9371.37
-9546.93 2975.93
8690.47 4947.29
-7615.94 6480.54
-9963.44 -854.264
-526.162 9986.15
-9896.69 1433.69
7148.13 6993.16
-6071.59 7945.81
9960.64 886.374
5976.7 -8017.43
7697.66 6383.26
5119.63 -8590.08
-9447.78 3277.1
9550.39 -2964.8
3899.04 -9208.56
9762.61 -2165.96
9999.48 -102.01
3165.97 9485.6
9796.79 2005.71
9999 -141.693
-8306.95 -5567.28
-7514.78 -6597.59
1322.4 9912.18
-8887.57 4583.78
-9730.98 -2303.91
7083.81 -7058.3
-4722.5 7727.5
2448.77 7763
-4746.82 -2406.19
-2464.42 1211.94
-6363.45 -783.59
-9023.1 -2513.95
-4727.11 967.1
