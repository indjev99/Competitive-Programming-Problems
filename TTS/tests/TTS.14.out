23678.440251
